src/test/regress/sql/function_extensions.sql

```diff
@@ -298,5 +298,13 @@ execute on initplan;
 explain select array(select f from hello_initplan() as f);
 select array(select f from hello_initplan() as f);
 
-
+-- Test INITPLAN functions with projections
+DROP TABLE IF EXISTS t8_function_scan;
+-- should succeed and add 3 rows with IDs (112, 223, 334)
+CREATE TABLE t8_function_scan AS SELECT country_id+1 AS a, country FROM get_country();
+-- should succeed and add 3 rows with IDs (113, 224, 335)
+INSERT INTO t8_function_scan SELECT country_id+2 AS a, country FROM get_country();
+-- should have 6 rows without any NULL entries
+SELECT * FROM t8_function_scan ORDER BY a;
+DROP TABLE t8_function_scan;
```

Comment:
Should we also remove get_country() function?

Reply:
That's not the only function that's not dropped. Should I just add cleanup for the whole file?

Reply:
I'm not sure. I'd rather just clean up after myself.

Reply:
Then probably not get_country(), since it wasn't created in this patch

Reply:
ok